### ISSUE TITLE
WISYWIG Toolbar doesn't grow anymore when there are many tools

### ISSUE THREAD

**Issue by MiniDigger** [directus/directus#3784]
Updated from rc28 to rc31:

wisywig in a repeater:
![image](https://user-images.githubusercontent.com/2185527/105500263-f5ab7f00-5cc2-11eb-8f6a-f01ab552058c.png)

`tox-toolbar__primary` has a height of 40px set.

```css
.tox .tox-toolbar-overlord .tox-toolbar__primary {
	height: unset;
}
```
in custom css fixes it for now
(first selectors are needed to override, `.tox .tox-toolbar, .tox .tox-toolbar__overflow, .tox .tox-toolbar__primary` is the selector that sets the height)

**Comment by MiniDigger** [comment]
offending commit https://github.com/directus/directus/commit/2aeafb2bce63d08cfdd4d02cb1261309b1659968#diff-fbcc7a77139bbea9064045bffbe51bcfdbbc21b90f3ed5addc73c66530ff7892R113

maybe min-height: 40px instead? I dont have a chance to test the sideeffects of that rn

**Comment by rijkvanzanten** [comment]
Fixed in 5a050a22db38ae71698459604ecc5fbbba5b1eb7
